### PR TITLE
Teststatus

### PR DESCRIPTION
**Pull request by KaiSzuttor**

### CODE REVIEW

**Comment by KaiSzuttor** [comment]
not yet finished, will wait for #26

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -69,3 +70,14 @@ ubuntu-python3-test:
...
+  stage: status
+  image: ubuntu:xenial
+  script: bash maintainer/gh_post_status.sh success
+  when: on_success
+
+status_failure:
+  stage: status
+  image: ubuntu:xenial
+  script: bash maintainer/gh_post_status.sh failure
+  when: on_failure
```
Comment (mkuron) [inline .gitlab-ci.yml]: Needs
```
tags:
  - docker
```

Comment (KaiSzuttor) [reply .gitlab-ci.yml]: Why?

Comment (mkuron) [reply .gitlab-ci.yml]: Otherwise these jobs can end up on the Mac builders.

Comment (KaiSzuttor) [reply .gitlab-ci.yml]: The macOS machines don't accept jobs without tags

Comment (mkuron) [reply .gitlab-ci.yml]: Not in our current configuration. But if we ever have other non-Docker builders, we have a problem. Just make the dependency on Docker explicit by adding the tag.

**Comment by KaiSzuttor** [comment]
it seems like the espressomd/bot team needs access to the docker repo @fweik, can you please add the docker repo for the team

**Comment by mkuron** [comment]
Oh no, @espresso-ci has become self-aware and is writing comments...

File: .gitlab-ci.yml (hunk 1 above)
Comment (KaiSzuttor) [reply .gitlab-ci.yml]: oh no, you think of windows build machines??? NOOOOOOO

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -72,12 +72,16 @@ ubuntu-python3-test:
 
 status_success:
   stage: status
-  image: ubuntu:xenial
+  image: $CI_REGISTRY/ubuntu:latest
```
Comment (mkuron) [inline .gitlab-ci.yml]: it's $CI_REGISTRY/espressomd/docker/ubuntu:latest

**Comment by KaiSzuttor** [comment]
Im debugging...as the commit names indicate :smile:

**Pull request merged**